Twits/Views/Twit/Edit.cshtml

```diff
@@ -0,0 +1,41 @@
+﻿@model Twits.Models.Twit
+
+@{
+    ViewBag.Title = "Edit";
+}
+
+<h2>Edit</h2>
+
+@using (Html.BeginForm())
+{
+    @Html.AntiForgeryToken()
```

Comment:
Again, don't forget to add a `ValidateAntiForgeryToken` attribute to the POST action method.